common/templates/darklang/preview_lang_include.html

```diff
@@ -0,0 +1,34 @@
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%page expression_filter="h"/>
+
+<h1>
+    ${_("Preview Language Setting")}
+</h1>
+<div >
+    <form class="form" action="/update_lang/" method="post">
+        <fieldset class="form-group">
+            <div class="field">
+                <label class="field-label">${_("Language Code")}
+                    <input class="field-input input-text" type="text" id="input_language" name="preview_lang"
+                       placeholder="e.g. en" aria-describedby="Input abbreviation for your language of choice" />
+                </label>
+            </div>
+        </fieldset>
+        <div class="form-actions">
+            <button class="btn-brand btn-base" type="submit" name="set_language" value="set_language">${_("Submit")}</button>
+            <button class="btn-brand btn-base" type="submit" name="reset" value="reset">${_("Reset")}</button>
+        </div>
+
+        <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }"/>
+    </form>
+
+    <br/>
+    % if not form_submit_message is UNDEFINED:
+        <h3 class="alert-title"> ${_(form_submit_message)}</h3>
```

Comment:
The translation shouldn't be happening here-- form_submit_message itself should already be translated.